docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:xenial
+
+MAINTAINER Herman Junge <herman.junge@consensys.net>
+
+RUN apt-get update -y \
+    && apt-get upgrade -q -y \
+    && apt-get dist-upgrade -q -y \
+    && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 923F6CA9 \
+    && echo "deb http://ppa.launchpad.net/ethereum/ethereum/ubuntu xenial main" \
+       | tee -a /etc/apt/sources.list.d/ethereum.list \
+    && apt-get update -q -y \
+    && apt-get install -y automake build-essential git-core libffi-dev \
+       libgmp-dev libssl-dev libtool pkg-config python-dev python-pip solc \
+    && pip install --upgrade pip \
+    && rm -rf /var/lib/apt/lists/* \
+    && git clone https://github.com/ethereum/pyrlp /apps/pyrlp \
```

Comment:
Adding `--depth=1` to all those clones would make the build a lot faster (also we don't need the history).